bin/compile-requirements.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+set -exo pipefail
+
+export CUSTOM_COMPILE_COMMAND="make compile-requirements"
+
+# We need this installed, but we don't want it to live in the main requirements
+# We will need to periodically review this pinning
+pip install --upgrade pip-tools==6.4.0  # needs at least this version to build
+pip install pip-compile-multi
```

Comment:
> Does this take very long? 
It takes a minute or so, as it'll often be in the pip cache as it doesn't change

> Did we want to use --upgrade on the pip-compile-multi as well?
We could do, but we might find that things drift more than expected when adding a new dependency and recompiling. Let's add it and see if it gets in our way

- [ ] @stevejalim `--upgrade` pip-compile-multi, too

Reply:
Is there anything special that was done here so that this has the permissions to install this within the container? On nucleus I'm getting a permission error b/c the user can't write to the file system.

Reply:
@robhudson No, didn't do anything special for Bedrock. I wonder if nucleus doesn't mount the root dir but maybe selectively mounts the ones it needs as container volumes?

Reply:
probably trying to do it after it switches to another user.

Reply:
oh yeah, this isn't during the image build... if the image is run as a non-root user it probably won't have perms.